docs/features/pouch_with_runV.md

```diff
@@ -136,7 +135,7 @@ It turns out that in experiment above kernel in host physical machine is 4.4.0-1
 
 ## Run legacy kernels
 
-runV(now kataconatiners) provides a general way to provide an isolated Linux kernel still based on OCI-compatible images. To be honest, Linux kernel running in Guest OS provisioned by runV is quite advanced and new. However, how to make legacy Linux kernel run in Guest OS is still a really huge challenge for the industry when using runV.
+runV(now kata containers) provides a general way to provide an isolated Linux kernel still based on OCI-compatible images. To be honest, Linux kernel running in Guest OS provisioned by runV is quite advanced and new. However, how to make legacy Linux kernel run in Guest OS is still a really huge challenge for the industry when using runV.
```

Comment:
since the icon shows us `katacontainers`, could we keep it unchanged? thanks

Reply:
Sure. Still fixes a typo (conatiners -> containers) though.